content/guides/migrating-to-cypress/protractor.md

```diff
@@ -0,0 +1,872 @@
+
+The Test Runner also helps you to write your tests by making it as easy as possible to find the right CSS selectors for the DOM elements in your application with its [Selector Playground](https://docs.cypress.io/guides/core-concepts/test-runner#Selector-Playground). The Selector Playground helps you cut down on time spent finding the right selectors so you can focus on what's important: writing tests that verify your app's logic.
+
+<DocsVideo src="/img/guides/migrating-to-cypress/DevTools.mp4" title="interacting with tests in a browser"></DocsVideo>
+
+### Faster feedback loops
+
+When it comes to your end-to-end tests, being able to see your tests as they run is critical to allowing you to confidently iterate faster. With Cypress, your tests are automatically re-run upon file save as you are iterating on them.
+
+Having your code editor and application under test within a browser side-by-side (shown below) while re-running tests on save is a highly productive workflow. It provides an instant feedback loop that allows you to iterate faster with confidence.
+
+<DocsVideo src="/img/guides/migrating-to-cypress/Auto-reloading.mp4" title="auto-reloading"></DocsVideo>
+
+### Time travel through tests
+
+The Cypress Test Runner gives you time travel capabilities to see exactly how your app was behaving at any point during test execution. Cypress takes DOM snapshots of your application under test as the Test Runner executes the commands and assertions in your tests. This enables you to view the **real UI** of your application at any point during your tests' execution. By clicking from one command to another in the [command log](/guides/core-concepts/test-runner#Command-Log), you can see which elements Cypress acted upon and how your application responded to the simulated **real user behavior**.
+
+<DocsVideo src="/img/guides/migrating-to-cypress/interactivity.mp4" title="Time travel debugging"></DocsVideo>
+
+## Getting Started
+
+### Recommended Installation
+
+We recommend using our [official Cypress Angular schematic](https://github.com/cypress-io/cypress/tree/master/npm/cypress-schematic#readme) to add Cypress to your Angular project:
```

Comment:
@matthamil do we want to update the schematic link to go to npm? (https://www.npmjs.com/package/@cypress/schematic)

Reply:
Can do!

Reply:
My only thought on linking to the `npm` page is that the following links in this paragraph (i.e. links to [this](https://github.com/cypress-io/cypress/tree/master/npm/cypress-schematic#running-the-builder-with-a-specific-browser) and [this](https://github.com/cypress-io/cypress/tree/master/npm/cypress-schematic#recording-test-results-to-the-cypress-dashboard)) can't be directly linked to on the `npm` page since those headers lack an ID. Those links don't really need to be consistent though.

Reply:
I'll just change the first link to the npm page and keep the others in this paragraph the same